vertical-pod-autoscaler/hack/vpa-process-yamls.sh

```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit
```

Comment:
The [no]errexit strategy is confusing to me, especially in nested scripts calling scripts. I would be glad to have it set always and instead use "|| true" on the commands that we actually expect to fail and are ok with that.

Reply:
Redinements applied.